texastribune/styleguides
# All CSS to Sass & clean out base styles

#### What's the plan?

The plan is to get rid of the base.css file and have all styles written in Sass. Also to clean out reset/normalize styles.
#### What's this PR do?

This PR adds a styleguide directory to the _sass with a main app.sass file that imports _base.sass, which includes general layout and grid styles the styleguides is using; _festivalstyle.sass, which includes styles specific to the festival styleguide and not the festival site; and _tribtalkstyle.sass, which includes styles specific to the TribTalk styleguide and not the TribTalk site. It cleans out reset and normalize styles that were in base.css and does not include them in _base.sass. It also adds styleguide Sass to the Gruntfile and tasks.
#### How should this be manually tested?

To manually test this PR, check that the build tasks are working as they should by running:
`grunt`

Then run styleguides locally with:
`make serve`

Check that the styleguide homepage, TribTalk page, and TTF14 style page look as they should (nothing glaringly unstyled or odd-looking is the main concern) by clicking around and checking out elements.
#### What are the relevant tickets?

This addresses Issue #3.
#### Smells?

Share any you see
#### TODOs:
- [x] Bring in the festival site HTML so this can be tested with this styleguide.min.css too
#### Technical debt note

Gets rid of reset/normalize styles and brings everything over to be written in Sass for styleguides, so should be a plus here.
